build/Makefile

```diff
@@ -102,20 +97,19 @@ printvar:
 	$(_hide_cmds)$(foreach d,$(DRIVERS),$(call per_driver_variable_print,$(d),$(VAR)) && ) echo
 
 
-# Minimal makefile for Sphinx documentation
-#
 PYTHON_CMD ?= python3
 
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = $(PYTHON_CMD) -msphinx
-SPHINXPROJ    = NIModularInstrumentsPythonAPI
-SOURCEDIR     = $(ROOT_DIR)/docs
-BUILDDIR      = $(BIN_DIR)/docs
+.PHONY: help Makefile
 
-html:
-	$(_hide_cmds)$(call log_command,$(SPHINXBUILD) -M html "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS))
 
-.PHONY: help Makefile
+# Any step that any driver build does that would invalidate unit testing, flake8 or generated html
+# needs to delete this file. This will trigger a tox run.
```

Comment:
Isn't the normal way to do this with GNU Make is to add anything that invalidates these things be a dependency of $(TOX_RUN_DONE) ?

Reply:
Yes, that would be the normal way. In order to do that, we would need to somehow dynamically generate that from the driver builds so that the list would be available here.